spec/src/main/asciidoc/release-notes.asciidoc

```diff
@@ -14,20 +14,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-## Release Notes
+[[release_notes_12]]
+== Release Notes for MicroProfile JWT 1.2
 
-### Changes in 1.1.1-RC2
+A full list of changes delivered in the 1.2 release can be found at link:https://github.com/eclipse/microprofile-jwt-auth/milestone/7[MicroProfile JWT 1.2 Milestone].
+
+=== API Changes
+
+- Convenience method has been added to allow retrieving claims from `JsonWebToken` by using the `Claims` enum (link:https://github.com/eclipse/microprofile-jwt-auth/issues/154[#154])
+
+=== Spec Changes
+
```

Comment:
Can you also add the link to the issue for the following items?

Reply:
@Emily-Jiang thanks, added and verified the links